src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IInterpolatedStringExpression.cs

```diff
@@ -0,0 +1,295 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public partial class IOperationTests : SemanticModelTestBase
+    {
+        [Fact, WorkItem(18300, "https://github.com/dotnet/roslyn/issues/18300")]
+        public void InterpolatedStringExpression_Empty()
+        {
+            string source = @"
+using System;
+
+internal class Class
+{
+    public void M()
+    {
+        Console.WriteLine(/*<bind>*/$""""/*</bind>*/);
+    }
+}
+";
+            string expectedOperationTree = @"
+IInterpolatedStringExpression (OperationKind.InterpolatedStringExpression, Type: System.String) (Syntax: '$""""')
```

Comment:
We should probably open a design issue to confirm that this is what we expect.

---
In reply to: [117338430](https://github.com/dotnet/roslyn/pull/19590#discussion_r117338430) [](ancestors = 117338430)